templates/macros/https_params.conf

```diff
@@ -0,0 +1,23 @@
+{% macro params() %}
```

Comment:
We could probably use this macro in `templates/include/https_params.conf` too, right? No need to duplicate the content, we just need to make sure `{{ nginx.prefix.config }}/include/https_params.conf` still works as expected.

Reply:
I was not sure about this, so I kept them seperated, but we could try